File: fixtures/code-graph/routes/typescript/near-matches.ts
const app = createUnrelatedClient();
const path = "/looks-like-a-route";

app.get(path, handler);
client.post("/also-not-a-route", handler);

const arbitrary = {
  path: "/not-a-router-config",
  component: LooksLikeAComponent,
};

File: fixtures/code-graph/routes/python/near_matches.py
from fastapi import FastAPI

app = FastAPI()
PREFIX = "/computed"


def route(path):
    return lambda function: function


@route("/not-a-framework-route")
def ordinary_decorator():
    return None


URL = "/also-not-a-route"


def computed(prefix, app, handler):
    app.route(prefix + "/dynamic")(handler)


@app.get(PREFIX + "/decorator")
def computed_decorator():
    return None
